documentation/snip-kafka-versions.sh

```diff
@@ -25,7 +25,7 @@ cat <<EOF
 EOF
 for i in "${!versions[@]}"
 do 
-    echo "| ${versions[$i]} | ${protocols[$i]} | ${formats[$i]} | 3.4.13"
+    echo "| ${versions[$i]} | ${protocols[$i]} | ${formats[$i]} | 3.4.14"
```

Comment:
As noted on the file this script generates, it's not as simple as this.

Reply:
Yeah 2.2.1 has 3.4.13 and 2.3.0 has 3.4.14. This issue is solved in the ZK upgrade PR as the versions are in the `kafka-versions.yaml` and so this script parses them correctly. We could put an if statement in, but that would be messy. Is it worth changing this? Or adding a note?

Reply:
I have patched in the zookeeper version features from my zookeeper upgrade branch so this should be solved now.